src/test/rustdoc/issue-15347.rs

```diff
@@ -1,15 +0,0 @@
-// Copyright 2105 The Rust Project Developers. See the COPYRIGHT
-// file at the top-level directory of this distribution and at
-// http://rust-lang.org/COPYRIGHT.
-//
-// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
-// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
-// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
-// option. This file may not be copied, modified, or distributed
-// except according to those terms.
-
-// compile-flags:--no-defaults --passes collapse-docs --passes unindent-comments
-
-// @has issue_15347/fn.foo.html
-#[doc(hidden)]
-pub fn foo() {}
```

Comment:
I don't think this test should be deleted. From #15347 it seems that these compile flags are supposed to result in this file existing, *with actual content* (whereas this PR is removing empty files for hidden docs).